src/components/layout/TransactionComplete/TransactionComplete.tsx

```diff
@@ -18,37 +18,29 @@ export enum transaction_types {
   BUY_ARNS_NAME = 'Buy ArNS Name',
 }
 
-function TransactionSuccess({
+function TransactionComplete({
   transactionId = new ArweaveTransactionID(STUB_ARWEAVE_TXID),
   state,
+  antId,
 }: {
   transactionId?: ArweaveTransactionID;
-  state: ANTContractJSON;
+  state?: ANTContractJSON; // for create ant cases
+  antId?: ArweaveTransactionID;
 }) {
   const [{}, dispatchGlobalState] = useGlobalState(); // eslint-disable-line
 
   return (
     <>
       <div className="flex-column center" style={{ gap: '3em' }}>
         <div className="flex-column center" style={{ gap: '2em' }}>
+          {/* TODO: configure error or fail states */}
           <AntCard
             domain={''}
             showTier={false}
-            id={transactionId}
-            state={state}
+            id={antId}
             compact={false}
             enableActions={false}
-            overrides={{
-              targetId: state.records['@'].transactionId,
-              ttlSeconds: state.records['@'].ttlSeconds,
-            }}
-            disabledKeys={[
-              'tier',
-              'evolve',
-              'maxSubdomains',
-              'domain',
-              'leaseDuration',
-            ]}
+            state={state}
           />
```

Comment:
<img width="775" alt="image" src="https://user-images.githubusercontent.com/12800001/233532780-cb2e18a0-4695-41e0-a20a-6808cfff182f.png">

after successfully deploying a name, I'm not seeing what I would expect for `domain` and `leaseDuration`